ivadomed/main.py

```diff
@@ -263,8 +272,9 @@ def run_segment_command(context, model_params):
             fname_img = bids_df.df[bids_df.df['filename'] == subject]['path'].to_list()
 
         # Add film metadata to options for segment_volume
-        if 'film_layers' in model_params and any(model_params['film_layers']) and model_params['metadata']:
-            metadata = bids_df.df[bids_df.df['filename'] == subject][model_params['metadata']].values[0]
+        if ModelParamsKW.FILM_LAYERS in model_params and any(model_params[ModelParamsKW.FILM_LAYERS]) \
+                and model_params[ModelParamsKW.METADATA]:
+            metadata = bids_df.df[bids_df.df['filename'] == subject][model_params[ModelParamsKW.METADATA]].values[0]
             options['metadata'] = metadata
```

Comment:
Here `options` are the same one I mentioned about dictionary used during run segmentation commands